File: MasterLogicaProgramacion/EjerciciosJavascript/Ej13.js
function dividirArray(array, tamArray) {
    if (array.length <= tamArray){
        return [array];
    }

    let arrayRetorno = [];
    let subArr = [];
    for (let i = 0; i < array.length; i++) {
        subArr.push(array[i]);

        if (subArr.length == tamArray) {
            arrayRetorno.push(subArr);
            subArr = [];
        }
    }

    if (subArr.length < tamArray && subArr.length != 0) {
        arrayRetorno.push(subArr);
    }

    return arrayRetorno;
}

console.log(dividirArray([7,8,9,10,11], 2));

File: Cursos/MasterLogicaProgramacion/EjerciciosPython/Ej13.py
def dividirArray(array, tamArray):
    
    if (len(array) <= tamArray):
        return [array]

    arrayRetorno = []
    subArr = []
    
    for i in range(0, len(array)):
        subArr.append(array[i])

        if (len(subArr) == tamArray):
            arrayRetorno.append(subArr)
            subArr = []

    if (len(subArr) < tamArray and len(subArr) != 0):
        arrayRetorno.append(subArr)

    return arrayRetorno

print(dividirArray([7,8,9,10,11], 2))